target/linux/kirkwood/profiles/00-default.mk

```diff
@@ -7,7 +7,7 @@
 
 define Profile/Default
 	NAME:=Default Profile
-	PACKAGES:=kmod-mwl8k swconfig wpad-basic
+	PACKAGES:=kmod-mwl8k wpad-basic
```

Comment:
This will break 4.19 and 4.14, so we should consider removing that right away ...

Reply:
I will send a patch for that later this week if nobody objects right away ...

Reply:
Both old kernels should be dropped. I agree.

Reply:
CHKDSK88: Would you provide an Acked-by for my removal patch then? Just here, so I can add it when I prepare the patch properly.

Reply:
Sure. 5.4 work without any known issue.
Drop 4.19 and 4.14 in kirkwood:
Acked-by: Pawel Dembicki <paweldembicki@gmail.com>